Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jekyll/jekyll
+Label MAINTAINER Amir Pourmand
```

Comment:
just wanted to double check: would you be OK maintaining the docker repo?

Reply:
That's OK.